static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -152,6 +172,45 @@ export function EventGraph({group, groupStats, searchQuery, event}: EventGraphPr
     []
   );
 
+  if (error) {
+    return (
+      <GraphAlert type="error" showIcon>
+        {error.message}
+      </GraphAlert>
+    );
+  }
+
+  if (isLoadingStats) {
```

Comment:
Loading state is great! I am noticing that the height changes by a couple pixels though, so the loading state is slightly different than the success state. Maybe we could use a set height to avoid this?

Reply:
thanks not sure what i did there, but "ExtraContent" should always be 114px\

![image](https://github.com/user-attachments/assets/d5c2767e-2d59-40c1-b4f0-8264c034084c)